classes/class-install.php

```diff
@@ -337,6 +337,7 @@ public function db_update_versions() {
 		$db_update_versions = array(
 			'3.0.0' /* @version 3.0.0 Drop the stream_context table, changes to stream table */,
 			'3.0.2' /* @version 3.0.2 Fix uppercase values in stream table, connector column */,
+			'3.0.8' /* @version 3.0.8 Increase size of user role IDs, user_roll column */,
```

Comment:
I think it is required. The `update` method runs through the versions returned by `db_update_versions` and then handles the update one at a time.  Unless I am missing a method somewhere that will call the update function.